eng/scripts/Create-APIView.ps1

```diff
@@ -86,6 +86,30 @@ function Get-ResourceProviderFromReadMePath {
     return $null
 }
 
+function Get-ImpactedTypespecProjects {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]$TypeSpecFile
+    )
+    $projectRootPaths = [System.Collections.Generic.HashSet[string]]::new()
```

Comment:
Do you really need to return a set from this function? This will always return a single project root since you are looking for a project path for the given file.